usr/share/rear/layout/save/GNU/Linux/240_swaps_layout.sh

```diff
@@ -46,7 +46,7 @@ Log "Saving Swap information."
         fi
 
         echo "swap $filename uuid=$uuid label=$label"
-    done < /proc/swaps
+    done <<< $(cat /proc/swaps | grep -v zram)
```

Comment:
I fear the current `grep -v zram` is too unspecific because
it removes all lines with a 'zram' (sub-)string snywhere.

https://github.com/rear/rear/issues/3343 indicates
a more specific string could be '/dev/zram' and
because /proc/swaps on my current workstation shows
```
Filename     Type         Size        Used    Priority
/dev/dm-1    partition    16775164    0       -2
```
it seems we are looking for '/dev/zram' only at the beginning
of the /proc/swaps lines so I would like to suggest
```
    # do not have /dev/zram* swap entries from disklayout.conf
    # see https://github.com/rear/rear/issues/3343
    done <<< $( grep -v '^/dev/zram' /proc/swaps )
```